arm-visualstudio/2014-04-01-preview/swagger/Csm.json

```diff
@@ -0,0 +1,870 @@
+        "operationId": "extensions_List",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "in": "path",
+            "name": "accountResourceName",
+            "description": "The name of the Visual Studio Team Services account resource.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The response contains all extension resource details for the specified account.",
+            "schema": {
+              "$ref": "#/definitions/ExtensionResourceGroupGetResponse"
+            }
+         }
+        },
+        "x-ms-examples": {
+            "getExtensionResources_List": {
+                "$ref": "../examples/GetExtensionResources_List.json"
+            }
+        }		
+      }
+    },    
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/microsoft.visualstudio/account/{accountResourceName}/extension/{extensionResourceName}": {
+      "put": {
+        "tags": [
+          "extensionResource"
+        ],
+        "summary": "Extensions_Create",
+        "description": "Registers the extension with a Visual Studio Team Services account.",
+        "operationId": "extensions_Create",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "An object containing additional information related to the extension request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ExtensionResourcePutRequest"
+            }
+          },
+          {
+            "in": "path",
+            "name": "accountResourceName",
+            "description": "The name of the Visual Studio Team Services account resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "extensionResourceName",
+            "description": "The name of the extension.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation succeeded. The extension resource was created for the specified account.",
+            "schema": {
+              "$ref": "#/definitions/ExtensionResourceGetResponse"
+            }
+         }
```

Comment:
If I understand correctly from the link you provided, it sounds like this tag should only be applied to async operations that provide a means of monitoring the request. That would not apply here; this is a synchronous operation that should only take a few seconds at most.